Core/include/Acts/Utilities/BinUtility.hpp

```diff
@@ -56,8 +56,8 @@ class BinUtility {
   ///
   /// @param bData is the provided binning data
   /// @param tForm is the (optional) transform
-  BinUtility(const BinningData& bData,
-             const Transform3& tForm = Transform3::Identity())
+  explicit BinUtility(const BinningData& bData,
+                      const Transform3& tForm = Transform3::Identity())
```

Comment:
is this breaking? since you also had to change some downstream code

Reply:
Technically yes, although I think `BinUtility` is not generally used from client code.